### ISSUE TITLE
Installing Troposphere does not get the latest changes to ECS

### ISSUE THREAD

**Issue by grasum** [cloudtools/troposphere#1190]
I am working on a ECS stack and I got errors for multiple properties. I found out that is because the latest ecs.py is not pulled when I upgrade troposphere or even install it from scratch. 

Example:
ContainerName and ContainerPort from ecs.ServiceRegistry were missing.
SchedulingStrategy from ecs.Service was missing

**Comment by oussemos** [comment]
Hey @grasum 

Could you please reproduce the steps like below
There's maybe an error in your imports

```
╰─# python
Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from troposphere.ecs import ServiceRegistry
>>> ServiceRegistry
<class 'troposphere.ecs.ServiceRegistry'>
```

**Comment by grasum** [comment]
Hi, that works well. The problem is that it had missing properties. I checked the package installation folder and ecs.py had the following :

```
class ServiceRegistry(AWSProperty):
    props = {
        'Port': (integer, False),
        'RegistryArn': (str, False),
    }
```
Missing ContainerName and ContainerPort.


Running a stack containing a Service resource
```
Service(
Name="service1",
ServiceRegistries=[
           ServiceRegistry(
                 ContainerName=container_name
           ),
      ],
)
```

Was returning an error:
AttributeError: ServiceRegistry object does not support attribute ContainerName. I also upgraded the package, but the classes were not changed.

**Comment by grasum** [comment]
I see. Thanks for the response !

**Comment by oussemos** [comment]
So, this issue can be closed.

**Comment by markpeek** [comment]
Looks like this was included in 2.3.4 and above. Closing.
